04/app.js

```diff
@@ -1,9 +1,33 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+	firstName: "Adam",
+	lastName: "Nowak",
+	born: {
+		day: "13",
+		month: "04",
+		year: "1985",
+	},
+};
+
+const today = new Date();
+console.log(today);
+
+const currentDay = today.getDate();
+console.log(currentDay);
+
+const currentMonth = today.getMonth() + 1;
+console.log(currentMonth);
+
+const birthday = function (object) {
+	const dayOfBirth = user.born.day;
```

Comment:
`this` używamy tylko wew. metod obiektu czyli jakby było `user.getInfo()` to wew. tego `.getInfo` można użyć `this`